packages/go/index.ts

```diff
@@ -172,7 +172,7 @@ Learn more: https://vercel.com/docs/runtimes#official-runtimes/go
     // Create cache so Go rebuilds fast with `vercel dev`
     // Old versions of the CLI don't assign this property
     const { devCacheDir = join(workPath, '.now', 'cache') } = meta;
-    goPath = join(devCacheDir, 'now-go', basename(entrypoint, '.go'));
+    goPath = join(devCacheDir, 'go', basename(entrypoint, '.go'));
```

Comment:
@TooTallNate Thoughts on this one? Kinda generic but its probably fine, right?

Reply:
I think it's fine. It's already within our `.vercel/cache` dir which we control.